lib/net.c

```diff
@@ -906,12 +917,10 @@ init_iface(struct gatekeeper_if *iface)
 	}
 
 	/* Initialize bonded port, if needed. */
-	if (iface->num_ports == 1)
+	if (iface->num_ports == 1 && iface->bonding_mode != BONDING_MODE_8023AD)
```

Comment:
The exact negation of `iface->num_ports > 1 || iface->bonding_mode == BONDING_MODE_8023AD` is `iface->num_ports <= 1 || iface->bonding_mode == BONDING_MODE_8023AD`. Could you edit all occurrencies?